Utilities/KWStyle/BuildKWStyle.cmake

```diff
@@ -22,8 +22,8 @@ set(CMAKE_C_COMPILER_LAUNCHER_FLAG -DCMAKE_C_COMPILER_LAUNCHER:FILEPATH=${CMAKE_
 if(NOT TARGET KWStyle)
   ExternalProject_Add(
     KWStyle
-    GIT_REPOSITORY "https://github.com/Kitware/KWStyle.git"
-    GIT_TAG d2cba46b3c63b5a7b6453052e85c593c48c0dffd
+    GIT_REPOSITORY "https://github.com/thewtex/KWStyle.git"
```

Comment:
The new SHA is now available in the official repository too, following the merge of https://github.com/Kitware/KWStyle/pull/112.